index.html

```diff
@@ -0,0 +1,329 @@
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <h2 class="connect-co">LET’S CONNECT</h2>
+
+        <ul class="social-media-icons">
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/github.png" alt="git" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/Linkedin.png" alt="linkedIn" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/angelite.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/twitter.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/medium.png" alt="image" />
+            </a>
+          </li>
+        </ul>
+        <div class="new-code">
+          <button type="submit" class="go-newmes">My resume</button>
+        </div>
+
+        <div class="skill">
+          <div class="event12">
+            <div class="event7">
+              <h3 class="languages">Languages</h3>
+              <button type="submit" class="dropdown">
+                <img src="./images/Icon-Chevron-Down-Larger.png" alt="" />
+              </button>
+            </div>
+            <ul class="language-list">
+              <li class="JavaScript1">
+                <div class="lang">
+                  <img src="./images/Ellipse-1.png" alt="" />
+                  <p>JavaSript</p>
+                </div>
+              </li>
+              <li class="HTML1">
+                <div class="lang">
+                  <img src="./images/Ellipse-2.png" alt="" />
+                  <p>HTML</p>
+                </div>
+              </li>
+              <li class="CSS1">
+                <div class="lang">
+                  <img src="./images/Ellipse-3.png" alt="" />
+                  <p>CSS</p>
+                </div>
+              </li>
+            </ul>
+          </div>
+          <div class="event-skill">
+            <div class="event18">
+              <h3>Frameworks</h3>
+              <button type="submit" class="dropdown">
+                <img src="./images/Disabled.png" alt="" />
+              </button>
+            </div>
+            <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+            <div class="event18">
+              <h3>Skills</h3>
+              <button type="submit" class="dropdown">
+                <img src="./images/Disabled.png" alt="" />
+              </button>
+            </div>
+            <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Good work implementing `<ul>` and `<li>` semantic tags to build your Languajes list ✔️. However, please kindly ensure to use `<ul>` and `<li>` semantic tags to create a list containing Languages, Skills, and Frameworks too, as requested by the previous reviewer 🤓.

Here is an idea of how the structure may look 👇🏽

```
<section class="skills-section">
  <ul>
    <li>
      <h3>Languages</h3>
      <ul>
        <li>
          Javascript
        </li>
        <li>
          HTML
        </li>
        <li>
          CSS
        </li>
      </ul>
    </li>

    <li>
      <h3>Frameworks</h3>
    </li>

    <li>
      <h3>Skills</h3>
    </li>
  </ul>
</section>
```

Reply:
ok. I do:

            <ul class="language-list">
              <li class="lang">
                <ul>
                  <li class="JavaScript1">
                    <img src="./images/Ellipse-1.png" alt="" />
                    <p>JavaSript</p>
                  </li>
                  <li class="HTML1">
                    <img src="./images/Ellipse-2.png" alt="" />
                    <p>HTML</p>
                  </li>
                  <li class="CSS1">
                    <img src="./images/Ellipse-3.png" alt="" />
                    <p>CSS</p>
                  </li>
                </ul>
              </li>
            </ul>